From a11fc3b16ac2aa06c240d2eefa8e712236735738 Mon Sep 17 00:00:00 2001
From: Yuwei Chen <yuwei.chen@intel.com>
Date: Mon, 3 Aug 2020 12:41:28 +0800
Subject: [PATCH] Test case2 for Change Source section

Description: Add a new source in Source section (Add the "Sec.h") in TestPkg/FakeEmuSec/Sec.inf.

Repo: edk2
Signed-off-by: Yuwei Chen <yuwei.chen@intel.com>
---
 TestPkg/FakeEmuSec/Sec.inf | 1 +
 1 file changed, 1 insertion(+)

diff --git a/TestPkg/FakeEmuSec/Sec.inf b/TestPkg/FakeEmuSec/Sec.inf
index 0a10d74c0b..7c5a54d28a 100644
--- a/TestPkg/FakeEmuSec/Sec.inf
+++ b/TestPkg/FakeEmuSec/Sec.inf
@@ -20,6 +20,7 @@
 
 [Sources]
   Sec.c
+  Sec.h
 
 [Sources.X64]
   X64/SwitchRam.asm
-- 
2.27.0.windows.1

